components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4947,12 +4954,6 @@ where
             .peer
             .mut_store()
             .maybe_gc_cache(alive_cache_idx, applied_idx);
-        if needs_evict_entry_cache(self.ctx.cfg.evict_cache_on_memory_ratio) {
-            self.fsm.peer.mut_store().evict_cache(true);
-            if !self.fsm.peer.get_store().cache_is_empty() {
-                self.register_entry_cache_evict_tick();
```

Comment:
Cache entries won't be evicted even exceeding evict_cache_on_memory_ratio? Any side effect if we remove this?

Reply:
This function is called very low frequently, which is not enough to avoid oom. So, remove it is no impact.

Reply:
This function is called very low frequently, which is not enough to avoid oom. So, remove it is no impact.